docs/en/getting_started/update.md

```diff
@@ -27,10 +27,10 @@ git remote -v
 
 again you should get:
 
-> upstream  git@github.com:consuldemocracy/consuldemocracy.git (fetch)\
-> upstream  git@github.com:consuldemocracy/consuldemocracy.git (push)\
 > origin  git@github.com:your_user_name/consuldemocracy.git (fetch)\
-> origin  git@github.com:your_user_name/consuldemocracy.git (push)
+> origin  git@github.com:your_user_name/consuldemocracy.git (push)\
+> upstream  git@github.com:consuldemocracy/consuldemocracy.git (fetch)\
+> upstream  git@github.com:consuldemocracy/consuldemocracy.git (push)
 
 ## Pulling changes from Consul Democracy
 
```

Comment:
Not really related to this line, but a few lines below we say:

```bash
git checkout -b upstream
```

Using the same name for a branch and an origin might lead to confusion when using git :thinking:. How about `git checkout -b upgrade`?